gammapy/spectrum/tests/test_models.py

```diff
@@ -159,3 +159,14 @@ def test_to_sherpa(spectrum):
     # test plot
     energy_range = [1, 10] * u.TeV
     model.plot(energy_range=energy_range)
+
+
+@requires_dependency('matplotlib')
+@requires_data('gammapy-extra')
+def test_table_model_from_file():
+    filename = '$GAMMAPY_EXTRA/datasets/ebl/ebl_franceschini.fits.gz'
+    energy_range = (0.03, 10)
```

Comment:
No need for local variables here IMO, since only used once.
Just pass as keyword arguments below.